Brokerages/InteractiveBrokers/InteractiveBrokersBrokerage.cs

```diff
@@ -142,8 +140,6 @@ private enum Region { America, Europe, Asia }
         // IB requests made through the IB-API must be limited to a maximum of 50 messages/second
         private readonly RateGate _messagingRateLimiter = new RateGate(50, TimeSpan.FromSeconds(1));
 
-        // used for reconnection attempts
-        private bool _previouslyInResetTime;
         // used to limit logging
         private bool _isWithinScheduledServerResetTimesLastValue;
```

Comment:
Should we add this flag into the state manager too?

Reply:
I didn't add this one in initially because it was only used for logging purposes, but I agree it can be moved as well, will do.